src/containers/DataGUI.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component, PropTypes } from 'react';
```

Comment:
I see.. I think you commented on the wrong line. Hence the confusion.
This duplication was necessary to use a different sass partial to add styles without affecting existing styles for `Metafields`. Yes, there should be a better way..